front/app/containers/Admin/projects/all/Lists/SortableFolderRow.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import { isNilOrError } from 'utils/helperUtils';
+
+// style
+import styled from 'styled-components';
+import { SortableRow } from 'components/admin/ResourceList';
+import useAdminPublications, {
+  IAdminPublicationContent,
+} from 'hooks/useAdminPublications';
+import ProjectFolderRow from '../../projectFolders/components/ProjectFolderRow';
+import { PublicationStatus } from 'services/projects';
+import useAuthUser from 'hooks/useAuthUser';
+import FolderChildProjects from './FolderChildProjects';
+
+const StyledSortableRow = styled(SortableRow)`
+  & .sortablerow-draghandle {
+    align-self: flex-start;
+  }
+`;
+interface Props {
+  key: string;
```

Comment:
The `key` prop is a special prop in React. Trying to access it will result in `undefined` being returned. We could rename it to something else.

Reply:
Yeah, we don't need to pass it as a prop. This is a remainder of the previous component. Removed it.